File: apps/front/src/entities/customer/lib/customer-type.ts
import { CompanyType } from '../model';

export function getCustomerTypeLocale(type: CompanyType): string {
  switch (type) {
    case CompanyType.Individual:
      return 'ИП';
    case CompanyType.Legal:
      return 'ООО';
    case CompanyType.Partner:
      return 'Партнер';
    case CompanyType.JointStock:
      return 'АО';
  }
}

File: apps/api/internal/customers/customer_type.go
package customers

type CustomerType string

const (
    CustomerTypeIndividual CustomerType = "individual" // Физ. лицо (разовый заказ)
    CustomerTypeLegal      CustomerType = "legal"      // Юр. лицо (постоянный клиент)
    CustomerTypePartner    CustomerType = "partner"    // Партнер (логистическая компания)
)
